test/unit/whitehall/exporters/documents_info_exporter_test.rb

```diff
@@ -0,0 +1,155 @@
+require 'test_helper'
+require "#{Rails.root}/lib/whitehall/exporters/documents_info_exporter"
+
+class DocumentsInfoExporterTest < ActiveSupport::TestCase
```

Comment:
I think these tests could be a bit more focused on what we're testing.

We should have tests that cover: locales, subtypes, and organisations.

locales seems to be covered by: https://github.com/alphagov/whitehall/pull/4901/files#diff-ed16559bfa573dce66e35a458a78c64cR105
subtypes is covered by: https://github.com/alphagov/whitehall/pull/4901/files#diff-ed16559bfa573dce66e35a458a78c64cR76
and orgs is covered by: https://github.com/alphagov/whitehall/pull/4901/files#diff-ed16559bfa573dce66e35a458a78c64cR125

I'm not really sure we need the tests at https://github.com/alphagov/whitehall/pull/4901/files#diff-ed16559bfa573dce66e35a458a78c64cR5 and https://github.com/alphagov/whitehall/pull/4901/files#diff-ed16559bfa573dce66e35a458a78c64cR47 as I'm not sure what functionality that's being tested.

Reply:
I've removed the two tests you've highlighted, these were needed originally when the exporter class was concerned about the different document types, this logic now resides in the controller so these tests as you've pointed out are pretty redundant.